### PR TITLE
Add new test (car)

### PR DESCRIPTION
**Pull request by AkininD**
Add new test (car)

### CODE REVIEW

**Review by vadim328 — APPROVED** [review]
Good!

**Review by vadim328 — APPROVED** [review]
Great!

File: test_myapi.py [hunk 1]
```diff
@@ -50,6 +50,19 @@ def test_predict_sandwich():
...
+    filename = "./tests/fixtures/car.jpeg"
+    response = client.post("/predict",
+                           files={"file": ("car.jpeg",
+                                           open(filename, "rb"),
+                                           "image/jpeg")})
+    predict_response = json.loads(response.content)
+    assert response.status_code == 200
+    assert any('car' in d for d in predict_response)
+    assert any('image_url' in d for d in predict_response)
+    tear_down()
```
Comment (Lemeri02) [inline test_myapi.py:63]: looks great!

**Review by Lemeri02 — APPROVED** [review]
Ok! LGFM!

**Pull request merged**